### PR TITLE
Adds Registered post tracking

### PR DESCRIPTION
**Pull request by krishnamuppaneni**
Added Registered post tracking. Needs bit of code cleaning. The date is parsed as MM-DD-YYYY and some time DD-MM-YYYY because of ambiguity. So I have set dayfirst flag as true so DD-MM-YYYY format will have precedence in an ambiguous date.

### CODE REVIEW

**Comment by captn3m0** [comment]
The indentation in your code is causing some issues. I've pushed an editorconfig. Try to make sure that you are following PEP8

File: web.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def robots():
 
 @route('/track/<id>')
 def track(id):
-    details = Tracker().track(id)
+    details = Tracker(id).track(id)
```
Comment (captn3m0) [inline web.py:22]: Don't like passing `id` in constructor as well as the track method. I think just passing it to track is cleaner.

Comment (krishnamuppaneni) [inline web.py:22]: The Tracker class constructor **init** needs id to determine the HOME_URL.

Comment (captn3m0) [inline web.py:22]: In that case, we can remove the argument from the `track` method?

Comment (krishnamuppaneni) [inline web.py:22]: Yeah that will work.

**Comment by captn3m0** [comment]
Closing this in favor of #5, where I've fixed the indentation and the id issues.

**Pull request closed without merging**